tailwind.config.js

```diff
@@ -18,9 +16,23 @@ module.exports = {
     theme: {
         extend: {
             colors: {
-                primary: colors.green[700],
-                secondary: colors.gray[400],
-            }
+                primary: 'rgb(var(--primary)  / <alpha-value>)', // Text color
+                secondary: 'rgb(var(--secondary) / <alpha-value>)', // Text inactive color
+                accent: 'rgb(var(--accent) / <alpha-value>)', // Theme color
+                enhanced: 'rgb(var(--enhanced) / <alpha-value>)', // Checkout conversion color
+                highlight: 'rgb(var(--highlight) / <alpha-value>)' // Background highlight color
+            },
+            borderColor: {
+                DEFAULT: 'rgb(var(--border) / <alpha-value>)',
+                border: 'rgb(var(--border) / <alpha-value>)'
+            },
+        },
+        fontFamily: {
+            sans: ['Arial', 'sans-serif']
+        },
+        container: {
+            center: true,
+            padding: '1.25rem',
```

Comment:
This is default tailwind, we're only using the center and padding so we don't have to add this when adding the 'container' class

Reply:
I think it's better not to use this. Please change it and maybe to reduce duplication create a container Blade component.

Reply:
So you want to wrap a container around content? 
For example you get this:
```
<container>
<div class="component ect">Content</div>
</container>
```

Instead of 
```
<div class="container component ect">
Content
</div>
```
So we add extra div's to wrap everything in a container instead of using only 1 class on the parent?

Reply:
Up to you, just don't use this tailwind config.